app/services/facebook_authenticator.rb

```diff
@@ -12,10 +12,14 @@ def initialize(auth_code)
   end
 
   def authenticate!
+    access_token  = get_access_token(@auth_code)
```

Comment:
best fix would be to just keep everything as it is and just don't pass @auth_code

> Am 08.09.2015 um 14:19 schrieb Oliver Azevedo Barnes <notifications@github.com>:
> 
> In app/services/facebook_authenticator.rb <https://github.com/simplabs/rails_api_auth/pull/18#discussion_r38918433>:
> 
> > @@ -12,10 +12,14 @@ def initialize(auth_code)
> >    end
> >  
> >    def authenticate!
> > +    access_token  = get_access_token(@auth_code)
> we could remove initialization and just have FacebookAuthenticator.authenticate!(auth_code) to fix that, but seems like too much work for little gain.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/simplabs/rails_api_auth/pull/18/files#r38918433>.
>